### PR TITLE
Refactored layout of source files

### LINKED ISSUE #72: Refactor source layout
The project currently has `GATTCharacteristic.swift` and `ATTProtocolDataUnit.swift` which contain over 20 types and over 2k LOC, as well as the HCI commands. We need to separate the project into one file per type (with nested types as exception). This source layout will prevent merge conflicts and make the project scalable for rapid development with a larger team, but at the cost of git history. There is also the worry of a limit of number for files, and longer compile times with Xcode and Swift Package Manager, but this approach is cleaner and scalable.

This should not affect the API, but we might need to rename some of the HCI commands since the Swift compiler has issues compiling nested types in extensions across different files based on the order of input files. This bug is not present when Whole Module Optimization is enabled.

### PR DESCRIPTION
**Pull request by colemancda**
**Issue**

Fixes #72 .

**What does this PR Do?**

Refactors the layout of the project's source files.

**Where should the reviewer start?**

`GATTCharacteristic.swift`
`ATTProtocolDataUnit.swift`

**Sweet giphy showing how you feel about this PR**

Giphy

### CODE REVIEW

**Comment by colemancda** [comment]
Result of [Integration 2](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/2b93d1ad85f64904a96cc4b1921b74fa)
---
*Duration*: 5 minutes and 42 seconds
*Result*: **Perfect build!** All 163 tests passed. :+1:
*Test Coverage*: 86%

**Comment by colemancda** [comment]
Result of [Integration 3](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/2b93d1ad85f64904a96cc4b19226d757)
---
*Duration*: 5 minutes and 45 seconds
*Result*: **Perfect build!** All 163 tests passed. :+1:
*Test Coverage*: 86%

**Comment by colemancda** [comment]
Result of [Integration 4](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/2b93d1ad85f64904a96cc4b19231b4a9)
---
*Duration*: 2 minutes and 46 seconds
*Result*: **1 error, failing state: build-errors**

**Comment by colemancda** [comment]
Result of [Integration 5](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/2b93d1ad85f64904a96cc4b19232081c)
---
*Duration*: 2 minutes and 40 seconds
*Result*: **1 error, failing state: build-errors**

**Comment by colemancda** [comment]
Result of [Integration 6](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/2b93d1ad85f64904a96cc4b192325848)
---
*Duration*: 2 minutes and 42 seconds
*Result*: **1 error, failing state: build-errors**

**Comment by colemancda** [comment]
Result of [Integration 7](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/2b93d1ad85f64904a96cc4b19232a869)
---
*Duration*: 2 minutes and 43 seconds
*Result*: **1 error, failing state: build-errors**

**Comment by colemancda** [comment]
Result of [Integration 8](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/2b93d1ad85f64904a96cc4b19232ddd6)
---
*Duration*: 2 minutes and 41 seconds
*Result*: **1 error, failing state: build-errors**

**Comment by colemancda** [comment]
Result of [Integration 9](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/2b93d1ad85f64904a96cc4b192333804)
---
*Duration*: 2 minutes and 43 seconds
*Result*: **1 error, failing state: build-errors**

**Comment by colemancda** [comment]
Result of [Integration 10](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/2b93d1ad85f64904a96cc4b192338b32)
---
*Duration*: 5 minutes and 25 seconds
*Result*: **1 error, failing state: build-errors**

**Comment by colemancda** [comment]
Result of [Integration 11](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/2b93d1ad85f64904a96cc4b19233d7bb)
---
*Duration*: 6 minutes and 10 seconds
*Result*: **Perfect build!** All 163 tests passed. :+1:
*Test Coverage*: 86%

**Comment by colemancda** [comment]
Result of [Integration 11](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/2b93d1ad85f64904a96cc4b19233d7bb)
---
*Duration*: 6 minutes and 10 seconds
*Result*: **Perfect build!** All 163 tests passed. :+1:
*Test Coverage*: 86%

**Comment by colemancda** [comment]
Result of [Integration 11](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/2b93d1ad85f64904a96cc4b19233d7bb)
---
*Duration*: 6 minutes and 10 seconds
*Result*: **Perfect build!** All 163 tests passed. :+1:
*Test Coverage*: 86%

**Comment by colemancda** [comment]
Result of [Integration 12](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/2b93d1ad85f64904a96cc4b1923fc3a6)
---
*Duration*: 6 minutes and 22 seconds
*Result*: **Perfect build!** All 163 tests passed. :+1:
*Test Coverage*: 86%

**Comment by colemancda** [comment]
Result of [Integration 12](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/2b93d1ad85f64904a96cc4b1923fc3a6)
---
*Duration*: 6 minutes and 22 seconds
*Result*: **Perfect build!** All 163 tests passed. :+1:
*Test Coverage*: 86%

File: Sources/ATTConnection.swift [hunk 1]
```diff
@@ -118,7 +118,7 @@ public final class ATTConnection {
         
         log?("Sending data... (\(sendOperation.data.count) bytes)")
         
-        try socket.send(Data(bytes: sendOperation.data))
```
Comment (colemancda) [inline Sources/ATTConnection.swift]: performance improvement

File: Sources/ATTConnection.swift [hunk 2]
```diff
@@ -310,7 +310,7 @@ public final class ATTConnection {
         // Retry for error response
         if opcode == .errorResponse {
             
-            guard let errorResponse = ATTErrorResponse(byteValue: [UInt8](data))
```
Comment (colemancda) [inline Sources/ATTConnection.swift]: performance improvement

File: Sources/ATTConnection.swift [hunk 3]
```diff
@@ -388,7 +388,7 @@ public final class ATTConnection {
             if type(of: notify).PDUType.attributeOpcode != opcode { continue }
             
             // attempt to deserialize
-            guard let PDU = foundPDU ?? type(of: notify).PDUType.init(byteValue: Array(data))
```
Comment (colemancda) [inline Sources/ATTConnection.swift]: performance improvement

File: Sources/ATTConnection.swift [hunk 4]
```diff
@@ -587,14 +587,14 @@ fileprivate final class ATTSendOperation {
         
         if opcode == ATT.Opcode.errorResponse.rawValue {
             
-            guard let errorResponse = ATTErrorResponse(byteValue: [UInt8](data))
```
Comment (colemancda) [inline Sources/ATTConnection.swift]: performance improvement

File: Sources/HCILEAddDeviceToWhiteList.swift [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    /// This command can be used at any time except when:
+    /// * any advertising filter policy uses the White List and advertising is enabled.
+    /// * the scanning filter policy uses the White List and scanning is enabled.
+    /// * the initiator filter policy uses the White List and a create connection command is outstanding.
+    ///
+    /// When a Controller cannot add a device to the White List because there is no space available,
+    /// it shall return the error code Memory Capacity Exceeded (0x07).
+    ///
+    /// Address is ignored when Address Type is set to 0xFF.
+    public struct HCILEAddDeviceToWhiteList: HCICommandParameter { //HCI_LE_Add_Device_To_White_List
```
Comment (colemancda) [inline Sources/HCILEAddDeviceToWhiteList.swift]: @carlos21 Rename to `HCILEAddDeviceToWhiteList` from `HCILowEnergyCommand.HCILEAddDeviceToWhiteList`. Remove extension.

File: Sources/HCILEAddDeviceToWhiteList.swift [hunk 6]
```diff
@@ -0,0 +1,52 @@
+//
+//  HCILEAddDeviceToWhiteList.swift
+//  Bluetooth
+//
+//  Created by Carlos Duclos on 6/14/18.
+//  Copyright © 2018 PureSwift. All rights reserved.
+//
+
+import Foundation
+
```
Comment (colemancda) [inline Sources/HCILEAddDeviceToWhiteList.swift]: @carlos21 Template:
```
// MARK - Method

public extension BluetoothHostControllerInterface {

     func lowEnergyAdd .... () { ... }
}

// MARK - Command

public struct HCILEAddDeviceToWhiteList: HCICommandParameter {
...
}
```

**Comment by colemancda** [comment]
Result of [Integration 12](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/2b93d1ad85f64904a96cc4b1923fc3a6)
---
*Duration*: 6 minutes and 22 seconds
*Result*: **Perfect build!** All 163 tests passed. :+1:
*Test Coverage*: 86%

**Comment by colemancda** [comment]
Result of [Integration 18](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/2b93d1ad85f64904a96cc4b19259db96)
---
*Duration*: 6 minutes and 53 seconds
*Result*: **Perfect build!** All 163 tests passed. :+1:
*Test Coverage*: 86%

**Comment by colemancda** [comment]
Result of [Integration 20](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/2b93d1ad85f64904a96cc4b19266fc6c)
---
*Duration*: 6 minutes and 55 seconds
*Result*: **Perfect build!** All 163 tests passed. :+1:
*Test Coverage*: 86%

**Comment by colemancda** [comment]
Result of [Integration 22](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/2b93d1ad85f64904a96cc4b19273ec7a)
---
*Duration*: 4 minutes and 14 seconds
*Result*: **1 error, failing state: build-errors**

**Comment by colemancda** [comment]
Result of [Integration 23](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/2b93d1ad85f64904a96cc4b192745dcd)
---
*Duration*: 4 minutes and 17 seconds
*Result*: **1 error, failing state: build-errors**

**Comment by colemancda** [comment]
Result of [Integration 24](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/2b93d1ad85f64904a96cc4b192748ed5)
---
*Duration*: 4 minutes and 18 seconds
*Result*: **1 error, failing state: build-errors**

**Comment by colemancda** [comment]
Result of [Integration 24](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/2b93d1ad85f64904a96cc4b192748ed5)
---
*Duration*: 4 minutes and 18 seconds
*Result*: **1 error, failing state: build-errors**

**Comment by colemancda** [comment]
Result of [Integration 27](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/2b93d1ad85f64904a96cc4b19275218c)
---
*Duration*: 7 minutes and 7 seconds
*Result*: **Perfect build!** All 163 tests passed. :+1:
*Test Coverage*: 86%

**Comment by colemancda** [comment]
Result of [Integration 28](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/2b93d1ad85f64904a96cc4b192823270)
---
*Duration*: 7 minutes and 17 seconds
*Result*: **Perfect build!** All 163 tests passed. :+1:
*Test Coverage*: 86%

**Comment by colemancda** [comment]
Result of [Integration 29](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/2b93d1ad85f64904a96cc4b1928f6104)
---
*Duration*: 7 minutes and 13 seconds
*Result*: All 163 tests passed, but please **fix 1 warning**.
*Test Coverage*: 86%

**Comment by colemancda** [comment]
Result of [Integration 30](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/2b93d1ad85f64904a96cc4b1929c962c)
---
*Duration*: 7 minutes and 14 seconds
*Result*: **Perfect build!** All 163 tests passed. :+1:
*Test Coverage*: 86%

**Comment by colemancda** [comment]
Result of [Integration 31](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/2b93d1ad85f64904a96cc4b192aa5c39)
---
*Duration*: 7 minutes and 27 seconds
*Result*: **Perfect build!** All 163 tests passed. :+1:
*Test Coverage*: 86%

**Comment by colemancda** [comment]
Result of [Integration 32](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/2b93d1ad85f64904a96cc4b192b8fb2f)
---
*Duration*: 7 minutes and 21 seconds
*Result*: **Perfect build!** All 163 tests passed. :+1:
*Test Coverage*: 86%

**Comment by colemancda** [comment]
Result of [Integration 33](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/2b93d1ad85f64904a96cc4b192c6cbbf)
---
*Duration*: 7 minutes and 34 seconds
*Result*: **Perfect build!** All 163 tests passed. :+1:
*Test Coverage*: 86%

**Comment by colemancda** [comment]
Result of [Integration 34](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/2b93d1ad85f64904a96cc4b192d4c55f)
---
*Duration*: 7 minutes and 38 seconds
*Result*: **Perfect build!** All 163 tests passed. :+1:
*Test Coverage*: 86%

**Comment by colemancda** [comment]
Result of [Integration 34](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/2b93d1ad85f64904a96cc4b192d4c55f)
---
*Duration*: 7 minutes and 38 seconds
*Result*: **Perfect build!** All 163 tests passed. :+1:
*Test Coverage*: 86%

**Comment by colemancda** [comment]
Result of [Integration 35](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/2b93d1ad85f64904a96cc4b192e35726)
---
*Duration*: 7 minutes and 30 seconds
*Result*: **Perfect build!** All 163 tests passed. :+1:
*Test Coverage*: 86%

**Comment by colemancda** [comment]
Result of [Integration 36](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/2b93d1ad85f64904a96cc4b192f1820e)
---
*Duration*: 7 minutes and 28 seconds
*Result*: **Perfect build!** All 163 tests passed. :+1:
*Test Coverage*: 86%

**Comment by colemancda** [comment]
Result of [Integration 37](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/054202d02a0d1e038383e0e594005357)
---
*Duration*: 7 minutes and 31 seconds
*Result*: **Perfect build!** All 163 tests passed. :+1:
*Test Coverage*: 86%

**Comment by colemancda** [comment]
Result of [Integration 38](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/054202d02a0d1e038383e0e5940e72e8)
---
*Duration*: 7 minutes and 37 seconds
*Result*: All 163 tests passed, but please **fix 1 warning**.
*Test Coverage*: 86%

**Comment by colemancda** [comment]
Result of [Integration 39](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/054202d02a0d1e038383e0e5941d7616)
---
*Duration*: 7 minutes and 37 seconds
*Result*: **Perfect build!** All 163 tests passed. :+1:
*Test Coverage*: 86%

**Comment by colemancda** [comment]
Result of [Integration 40](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/054202d02a0d1e038383e0e5942c1b72)
---
*Duration*: 1 minute and 31 seconds
*Result*: **1 error, failing state: build-errors**

**Comment by colemancda** [comment]
Result of [Integration 41](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/054202d02a0d1e038383e0e5942c63e6)
---
*Duration*: 1 minute and 31 seconds
*Result*: **1 error, failing state: build-errors**

**Comment by colemancda** [comment]
Result of [Integration 41](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/054202d02a0d1e038383e0e5942c63e6)
---
*Duration*: 1 minute and 31 seconds
*Result*: **1 error, failing state: build-errors**

**Comment by colemancda** [comment]
Result of [Integration 42](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/054202d02a0d1e038383e0e5942cbbfc)
---
*Duration*: 1 minute and 33 seconds
*Result*: **1 error, failing state: build-errors**

**Comment by colemancda** [comment]
Result of [Integration 42](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/054202d02a0d1e038383e0e5942cbbfc)
---
*Duration*: 1 minute and 33 seconds
*Result*: **1 error, failing state: build-errors**

**Comment by colemancda** [comment]
Result of [Integration 43](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/054202d02a0d1e038383e0e5942cff56)
---
*Duration*: 1 minute and 39 seconds
*Result*: **1 error, failing state: build-errors**

**Comment by colemancda** [comment]
Result of [Integration 44](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/054202d02a0d1e038383e0e5942d4cbc)
---
*Duration*: 1 minute and 39 seconds
*Result*: **1 error, failing state: build-errors**

**Comment by colemancda** [comment]
Result of [Integration 45](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/054202d02a0d1e038383e0e594403256)
---
*Duration*: 1 minute and 42 seconds
*Result*: **1 error, failing state: build-errors**

**Comment by colemancda** [comment]
Result of [Integration 46](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/054202d02a0d1e038383e0e59449e2f5)
---
*Duration*: 7 minutes and 51 seconds
*Result*: **Perfect build!** All 165 tests passed. :+1:
*Test Coverage*: 86%

File: Sources/ATTErrorResponse.swift [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+        self.init(request: request, attributeHandle: attributeHandle, error: error)
+    }
+    
+    public var data: Data {
+        
+        var bytes = Data(repeating: 0, count: type(of: self).length)
+        
+        bytes[0] = type(of: self).attributeOpcode.rawValue
+        bytes[1] = request.rawValue
+        bytes[2] = attributeHandle.littleEndian.bytes.0
```
Comment (carlos21) [inline Sources/ATTErrorResponse.swift]: @colemancda littleEndian is being called twice

File: Sources/ATTFindInformationResponse.swift [hunk 8]
```diff
@@ -0,0 +1,209 @@
...
+}
+
+// MARK: - Supporting Types
+
+public extension ATTFindInformationResponse {
+
+    public enum Format: UInt8 {
+        
+        /// A list of 1 or more handles with their 16-bit Bluetooth UUIDs.
+        case bit16      = 0x01
```
Comment (carlos21) [inline Sources/ATTFindInformationResponse.swift]: @colemancda why these spaces?

Comment (colemancda) [reply Sources/ATTFindInformationResponse.swift]: its a C convention, to have it aligned

File: Sources/ATTErrorResponse.swift (hunk 7 above)
Comment (colemancda) [reply Sources/ATTErrorResponse.swift]: I'll fix that, this was the first ATT message implemented xD

File: Sources/ATTFindInformationResponse.swift (hunk 8 above)
Comment (carlos21) [reply Sources/ATTFindInformationResponse.swift]: 🤢

File: Sources/ATTFindInformationResponse.swift [hunk 9]
```diff
@@ -0,0 +1,209 @@
...
+         case let (.bit16(lhsValue), .bit16(rhsValue)):
+         return lhsValue == rhsValue
+         
+         case let (.bit128(lhsValue), .bit128(rhsValue)):
+         return lhsValue == rhsValue
+         
+         default:
+         return false
+         }
+         }*/
```
Comment (carlos21) [inline Sources/ATTFindInformationResponse.swift]: @colemancda I think we should remove commented code or add a TODO If something is missing

File: Sources/ATTFindInformationResponse.swift (hunk 8 above)
Comment (colemancda) [reply Sources/ATTFindInformationResponse.swift]: Feel free to remove the spaces

**Comment by colemancda** [comment]
Result of [Integration 50](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/054202d02a0d1e038383e0e594a52087)
---
*Duration*: 7 minutes and 1 second
*Result*: **1 error, failing state: build-errors**

**Comment by colemancda** [comment]
Result of [Integration 51](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/2b93d1ad85f64904a96cc4b19200bf13/054202d02a0d1e038383e0e594a57d73)
---
*Duration*: 8 minutes and 6 seconds
*Result*: **Perfect build!** All 169 tests passed. :+1:
*Test Coverage*: 86%

**Pull request merged**